Can not run make test-integration

When running `make test-integration`, I got the following errors. Did I miss anything?

```
$ make test-integration
+++ [0828 17:19:43] Checking etcd is on PATH
/Users/klaus/Workspace/kubernetes/third_party/etcd/etcd
+++ [0828 17:19:43] Starting etcd instance
etcd --advertise-client-urls http://127.0.0.1:2379 --data-dir /var/folders/c3/x5_ln0h55lv6l13ncsl34n400000gn/T/tmp.nzTFdi9w --listen-client-urls http://127.0.0.1:2379 --debug > "/dev/null" 2>/dev/null
Waiting for etcd to come up.
+++ [0828 17:19:44] On try 2, etcd: : http://127.0.0.1:2379
{"action":"set","node":{"key":"/_test","value":"","modifiedIndex":4,"createdIndex":4}}
+++ [0828 17:19:44] Running integration test cases
Running tests for APIVersion: v1,authorization.k8s.io/v1beta1,autoscaling/v1,batch/v1,apps/v1alpha1,policy/v1alpha1,extensions/v1beta1,rbac.authorization.k8s.io/v1alpha1,certificates/v1alpha1
+++ [0828 17:19:50] Running tests without code coverage
can't load package: package k8s.io/kubernetes/test/integration/metrics: no buildable Go source files in /Users/klaus/Workspace/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/integration/metrics
make[1]: *** [test] Error 1
+++ [0828 17:19:50] Cleaning up etcd
+++ [0828 17:19:50] Integration test cleanup complete
make: *** [test-integration] Error 1
```

--- a/hack/.linted_packages
+++ b/hack/.linted_packages
@@ -208,6 +208,7 @@ test/images/serve_hostname
 test/integration/discoverysummarizer
 test/integration/examples
 test/integration/federation
+test/integration/metrics
 test/integration/openshift
 test/soak/cauldron
 test/soak/serve_hostnames

--- a/test/integration/metrics/doc.go
+++ b/test/integration/metrics/doc.go
@@ -1,0 +1,17 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics

